<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>757</width>
    <height>552</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">colors-light {
header: #FFFFFF; /**/
menu-top: #FAFAFA; /**/
menu-left: #EEEEEE; /**/
menu-hover: #E1E1E1; /**/
menu-hover-left-border: #616161; /**/
menu-text-select: #C53929; /**/
text: #555555 /*For QToolButton inactive*/
}

QToolBar {
    background:#EEEEEE;
    spacing: 12px; /* spacing between items in the tool bar */
	padding: 9px 9px 9px 9px;
 }

QToolBar::handle {
    image: url(handle.png);
 }

QToolButton {
	background: #EEEEEE;

	border-width: 3px;
	border-color: #616161;
	border-style: transparent;

	color: #555555;
	font-family: &quot;Garamond&quot;,Arial,Times;
	font-size: 16px;
	font-weight: bold;

	margin: 0px;
	padding: 0px;
}

QToolButton:hover {
	background: #E1E1E1;
	border-bottom-style: solid;

}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedCentral">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="pageFileViewer">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTabWidget" name="tabFileViewer">
          <widget class="QWidget" name="tab1">
           <attribute name="title">
            <string>Tab 1</string>
           </attribute>
          </widget>
          <widget class="QWidget" name="tab2">
           <attribute name="title">
            <string>Tab 2</string>
           </attribute>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pageSetupCamera">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label">
          <property name="autoFillBackground">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>lblSetupCamera</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>This is windowTitle</string>
   </property>
   <property name="movable">
    <bool>true</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionLogo"/>
   <addaction name="actionFile_Viewer"/>
   <addaction name="actionSetup_Camera"/>
   <addaction name="actionVerification"/>
   <addaction name="actionSettings"/>
   <addaction name="actionExit"/>
   <addaction name="actionUpdate_CSS"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionFile_Viewer">
   <property name="icon">
    <iconset>
     <normalon>:/icons/icons/appbar.cabinet.files.variant.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>File Viewer</string>
   </property>
  </action>
  <action name="actionSetup_Camera">
   <property name="text">
    <string>Setup Camera</string>
   </property>
  </action>
  <action name="actionVerification">
   <property name="text">
    <string>Verification</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionLogo">
   <property name="text">
    <string>Rhinoplassistant</string>
   </property>
  </action>
  <action name="actionUpdate_CSS">
   <property name="text">
    <string>Update CSS</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
